language/move-lang/src/parser/syntax.rs

```diff
@@ -1786,12 +1823,36 @@ fn parse_module(tokens: &mut Lexer<'_>) -> Result<ModuleDefinition, Error> {
             Tok::Spec => ModuleMember::Spec(parse_spec_block(tokens)?),
             Tok::Use => ModuleMember::Use(parse_use_decl(tokens)?),
             Tok::Friend => ModuleMember::Friend(parse_friend_decl(tokens)?),
-            Tok::Const => ModuleMember::Constant(parse_constant(tokens)?),
-            // TODO rework parsing modifiers
-            _ if is_struct_definition(tokens)? => {
-                ModuleMember::Struct(parse_struct_definition(tokens)?)
+            _ => {
```

Comment:
Do we also want to allow modifiers on any module member here instead of just on consts/functions/structs? If we want to support annotations with this, then I think we will? But you may want to make that change in the annotations PR. What do you think?

Reply:
Annotations are going to be separate from these modifiers. so for now I think the split is fine